game/Makefile

```diff
@@ -1,7 +1,7 @@
 CXX = g++
 CXXFLAGS = -std=c++14 -Wall -MMD
 EXEC = hackmon
-OBJECTS = action.o
+OBJECTS = action.o hackmon.o
```

Comment:
maybe remove the changes to the file so that they don't conflict? since they're not really necessary